lib/MainTemplate.js

```diff
@@ -194,7 +193,7 @@ module.exports = class MainTemplate extends Template {
 	}
 
 	renderAddModule(hash, chunk, varModuleId, varModule) {
-		return this.applyPluginsWaterfall("add-module", "modules[" + varModuleId + "] = " + varModule + ";", chunk, hash, varModuleId, varModule);
+		return this.applyPluginsWaterfall("add-module", `modules["${varModuleId}"] = ${varModule};`, chunk, hash, varModuleId, varModule);
```

Comment:
Here the same. `"` must be omited. `modules["${varModuleId}"]` -> `modules[${varModuleId}]`